subcommands/targets/offline-update.go

```diff
@@ -511,3 +537,35 @@ func signBundleTargets(rootMeta *client.AtsTufRoot, bundleTargetsMeta *tuf.Signe
 	bundleTargetsMeta.Signatures = append(bundleTargetsMeta.Signatures, signatures[0])
 	return nil
 }
+
+func doCheckBundle(cmd *cobra.Command, args []string) {
+	tufMetaPath := path.Join(args[0], "tuf")
+	bundleTufMeta, err := getBundleTargetsMeta(tufMetaPath, true)
+	subcommands.DieNotNil(err)
+	bundleMeta := ouBundleTufMeta{}
+	subcommands.DieNotNil(json.Unmarshal(*bundleTufMeta.Signed, &bundleMeta))
+	fmt.Println("Bundle targets info:")
+	fmt.Printf("\tType:\t\t%s\n", bundleMeta.ouBundleMeta.Type)
+	fmt.Printf("\tTag:\t\t%s\n", bundleMeta.Tag)
+	fmt.Printf("\tExpires:\t%s\n", bundleMeta.SignedCommon.Expires)
+	fmt.Println("\tTargets:")
+	for _, target := range bundleMeta.Targets {
+		fmt.Printf("\t\t\t%s\n", target)
+	}
+	fmt.Println("\tSignatures:")
+	for _, sig := range bundleTufMeta.Signatures {
+		fmt.Printf("\t\t\t%s\n", sig.KeyID)
```

Comment:
Do you want to consider this being a list like below?
```suggestion
		fmt.Printf("\t\t- %s\n", sig.KeyID)
```

The same on line 564.

Reply:
It is consistent with the output of the rest field values. I would rather do `"\t\t\t- %s\n" but not `"\t\t- %s\n"`.

Reply:
The `"\t\t\t- %s\n"` makes sense to me.
The point was to make it a bullet list.